tests/unit/api/BackbeatAPI.spec.js

```diff
@@ -0,0 +1,52 @@
+const assert = require('assert');
+
+const BackbeatAPI = require('../../../lib/api/BackbeatAPI');
+const BackbeatRequest = require('../../../lib/api/BackbeatRequest');
+const config = require('../../config.json');
+
+const fakeLogger = {
+    trace: () => {},
+    error: () => {},
+    info: () => {},
+    debug: () => {},
+};
+
+describe('BackbeatAPI', () => {
+    let bbapi;
+
+    before(() => {
+        bbapi = new BackbeatAPI(config, fakeLogger, { timer: true });
+    });
+
+    it('should validate routes', () => {
+        // valid routes
+        const valid1 = new BackbeatRequest({ url: '/_/metrics/crr/all' });
```

Comment:
For a more compact and less redundant code, instead of having multiple variables, you could have one array for the valid requests, and one for the invalid ones, and loop over the array items to validate with `assert.ok` or `assert.equal`